package.json

```diff
@@ -30,9 +30,8 @@
     "test-integration-mock": "./node_modules/.bin/mocha test/integration",
     "test-integration-no-mock": "./node_modules/.bin/mocha test/integration-no-mock/**/*.test.js",
     "test-integration": "npm run test-integration-mock && npm run test-integration-no-mock",
-    "coverage": "./node_modules/.bin/nyc --reporter=html --reporter=text ./node_modules/.bin/mocha test --NODE_CONFIG_DIR=./test/config",
```

Comment:
Why the removal?

Reply:
Running "npm audit" results in 1 critical vulnerability for "nyc" library - https://npmjs.com/advisories/755
Discussed with Dimo and removed the "nyc" library and the "coverage" script as it is no longer used.

Also, changed the "test-npm-security" script to match the one executed in Jenkins.